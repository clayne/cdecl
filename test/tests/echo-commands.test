EXPECTED_EXIT=0
cdecl -bEL$LINENO <<END
declare x as int
set noecho-commands
declare y as int
set echo-commands
declare z as int
END
