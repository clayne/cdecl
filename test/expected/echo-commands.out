cdecl> declare x as int
int x;
cdecl> set noecho-commands
int y;
cdecl> declare z as int
int z;
